<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopBar</class>
 <widget class="QMainWindow" name="TopBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>40</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>999</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>OpenBangla Keyboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #000000;
    border: 1px solid rgba(255, 255, 255, 0.1);
    border-radius: 10px;
}

QPushButton {
    background: transparent;
    border: none;
    border-radius: 10px;
    padding: 12px;
    margin: 2px;
    color: rgba(0, 0, 0, 0.7);
    font-size: 18px;
}

QPushButton:hover {
    background: rgba(255, 255, 255, 0.3);
}

QPushButton:pressed {
    background: rgba(255, 255, 255, 0.03);
}

QPushButton#buttonSetLayout:checked {
    background: rgba(14, 165, 233, 0.15);
    border: 1px solid rgba(14, 165, 233, 0.2);
    border-radius: 10px;
}

QPushButton#buttonShutdown:hover {
    background: rgba(255, 0, 0, 0.55);
    border-radius: 10px;
}
QWidget#centralwidget {
    background-color: #000000;
    border-radius: 10px;
}
QToolTip {
    background: #0f0f0f;
    color: rgba(255, 255, 255, 0.9);
    border: 1px solid rgba(255, 255, 255, 0.1);
    border-radius: 10px;
    padding: 8px 12px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>520</width>
     <height>80</height>
    </size>
   </property>
   <widget class="QPushButton" name="buttonIcon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>54</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Move</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSetLayout">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>0</y>
      <width>54</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Change Layout</string>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonViewLayout">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>0</y>
      <width>54</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>View information about the current layout</string>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSettings">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>0</y>
      <width>54</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Settings</string>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonShutdown">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>0</y>
      <width>54</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Quit</string>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>24</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>